src/main/java/seedu/address/model/datetime/WeeklyTimeslot.java

```diff
@@ -0,0 +1,147 @@
+package seedu.address.model.datetime;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.commons.util.AppUtil.checkArgument;
+
+import java.time.DayOfWeek;
+import java.time.LocalTime;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+import java.time.format.TextStyle;
+import java.util.Locale;
+import java.util.Objects;
+
+/**
+ * Represents a timeslot that repeats every week in ModQuik.
+ * Guarantees: immutable
+ */
+
+public class WeeklyTimeslot {
+    public static final String MESSAGE_CONSTRAINTS_TIMES =
+            "Times should be in HH:mm format, e.g. 08:00";
```

Comment:
There is a bit of confusion in message constraints here.

Reply:
Fixed!